config-model-api/src/test/java/com/yahoo/config/application/api/DeploymentSpecTest.java

```diff
@@ -276,8 +276,7 @@ public void productionSpecWithDuplicateRegions() {
             DeploymentSpec.fromXml(r);
             fail("Expected exception");
         } catch (IllegalArgumentException e) {
-            assertEquals("All declared regions must be unique, but found these duplicated regions: " +
-                                 "[us-west-1, us-central-1]", e.getMessage());
+            assertEquals("prod.us-west-1 is listed twice in deployment.xml", e.getMessage());
```

Comment:
Seems it no longer lists all duplicates. It fails to mention "prod.us-central-1" which appears in both `<parallel>` tags.

Reply:
Sure, but I don't think it's ever worth additional code to create more complete error messages like that: very small probability * very small added value

Reply:
Ok, test input should be simplified as well then, second `<parallel>` tag can be removed.

Reply:
ok, done